### PR TITLE
KEA-36 As a user, I want to know about the different groups so that I know what to get involved in

### PR DESCRIPTION
**Pull request by hyunghoseo**

### CODE REVIEW

File: frontend/src/screens/GroupsScreen.tsx [hunk 1]
```diff
@@ -1,16 +1,95 @@
...
+import { GroupList } from "@/components/GroupCard";
+import ScreenWrapper from "@/components/ScreenWrapper";
+
+export const GroupsScreen = () => {
+  const { data: { data: groups } = {}, isLoading, isError } = useGetGroups({});
+  const { t } = useTranslation();
+  const { h1, h2, bodyNormal, link } = useTypographyStyles();
+  const styles = useStyles();
+
+  const groupTypes: NonNullable<Group["Type"]>[] = [
```
Comment (doolri1276) [inline frontend/src/screens/GroupsScreen.tsx:21]: Could we get this from the database?

File: frontend/src/config/translations.ts [hunk 2]
```diff
@@ -28,6 +45,14 @@ export const translations: Translations = {
     "nav.Events": "이벤트",
     "nav.Announcements": "공지사항",
     "nav.ContactUs": "문의하기",
+    "page.Groups.title": "부서 및 동아리",
+    "page.Groups.description":
+      "뉴욕한인가정교회의 부서 및 동아리 대해 자세히 알아보세요. 각 그룹 또는 부서는 커뮤니티 내에서 역할을 합니다. 참여할 수 있는 그룹이나 궁금한 사항이 있으시면 언제든지 {link}.",
```
Comment (doolri1276) [inline frontend/src/config/translations.ts:50]: @jiahn326 You are good at writing. Could you review this to be more natural in Korean?

File: frontend/src/components/GroupCard/GroupList.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+export const GroupList: React.FC<GroupListProps> = ({
+  isLoading,
+  groups,
+  groupKey,
+}) => {
+  const styles = useStyles();
+
+  const { isMobile } = useResponsiveLayout();
+
+  const numColumns = isMobile ? 2 : 4;
```
Comment (doolri1276) [inline frontend/src/components/GroupCard/GroupList.tsx:24]: Can we go from 4, 3, and 2?

File: frontend/src/components/GroupCard/GroupCard.tsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+interface GroupCardProps extends Partial<Group> {
+  isLoading?: boolean;
+  style?: StyleProp<ViewStyle>;
+}
+
+export const GroupCard: React.FC<GroupCardProps> = (props) => {
+  const styles = useStyles();
+  const { h4, bodySmall } = useTypographyStyles();
+  const blurHash = "LBQ0w8og~Vt7xufQWCfQ_1j[9Hay";
```
Comment (doolri1276) [inline frontend/src/components/GroupCard/GroupCard.tsx:19]: Do we save the hash value here?

File: frontend/src/screens/GroupsScreen.tsx (hunk 1 above)
Comment (hyunghoseo) [reply frontend/src/screens/GroupsScreen.tsx:21]: it's a typescript type, so no, it only works during compile time. It's being retrieved from the types file that is generated from the API spec

File: frontend/src/components/GroupCard/GroupList.tsx (hunk 3 above)
Comment (hyunghoseo) [reply frontend/src/components/GroupCard/GroupList.tsx:24]: we can, we just need to determine a middle breakpoint. We can create a separate story to add this

File: frontend/src/screens/GroupsScreen.tsx (hunk 1 above)
Comment (doolri1276) [reply frontend/src/screens/GroupsScreen.tsx:21]: Gotcha!

File: frontend/src/components/GroupCard/GroupList.tsx (hunk 3 above)
Comment (doolri1276) [reply frontend/src/components/GroupCard/GroupList.tsx:24]: Sure. Could you create a separate story for this?

File: frontend/src/components/GroupCard/GroupCard.tsx (hunk 4 above)
Comment (hyunghoseo) [reply frontend/src/components/GroupCard/GroupCard.tsx:19]: what do you mean? if you mean extracting it for reuse in other files, we could but it's only being used in 1 file right now

File: frontend/src/assets/placeholder.jpg
[the remark this answers is not in the document]
Comment (hyunghoseo) [reply frontend/src/assets/placeholder.jpg:1]: we can change to png and add the x2 and x3 images if you want

[the remark this answers is not in the document]
Comment (doolri1276) [reply frontend/src/assets/placeholder.jpg:1]: Let's do that to keep the image resolutions :)

File: frontend/src/components/GroupCard/GroupCard.tsx (hunk 4 above)
Comment (doolri1276) [reply frontend/src/components/GroupCard/GroupCard.tsx:19]: Yes, there is 1 file right now. This is the hash value to be used on other pages too? If yes, then would it be better for you to set it up now? The others including me could be not sure where to put it.

Comment (hyunghoseo) [reply frontend/src/components/GroupCard/GroupCard.tsx:19]: sure thing

File: frontend/src/config/translations.ts (hunk 2 above)
Comment (doolri1276) [reply frontend/src/config/translations.ts:50]: page.Groups.description : 뉴욕한인가정교회의 부서 및 동아리에 대해 자세히 알아보세요. 각 부서 및 그룹들은 교회에 기여하고 다양한 관심사로 커뮤니티에 문화 교류의 장을 마련합니다. 더 많은 그룹에 참여하고 싶으시거나 궁금한 점이 있으신가요? {link}.
linkText : 문의해주세요

**Review by doolri1276 — APPROVED** [review]

**Pull request merged**